cypress/elements/dashboardItem.js

```diff
@@ -0,0 +1,15 @@
+export const chartSel = '.highcharts-container'
+export const chartSubtitleSel = '.highcharts-subtitle'
+export const chartXAxisLabelSel = '.highcharts-xaxis-labels'
+export const mapSel = '.dhis2-map-plugin'
+export const tableSel = '.pivot-table-container'
+
+export const itemDetailsSel = '[data-test="dashboarditem-footer"]'
+
+export const getDashboardItem = itemUid =>
```

Comment:
would probably be good to differentiate these chainable functions (that returns a `cy` object) in a way that makes it clear that they are indeed chainable. I have the same issue in DV, but haven't come up with a good way to do it yet.

However in this case, where all the function does is `cy.get(id)` I think it would be better to not use a function, as it doesn't really add much functionality. If this is ok here then really it should be done for all `Sel`'s above too 🤷‍♂️  Or is this because of the concat selector `dashboarditem-${itemUid}`? If the `itemUid` is truly unique you could just a wildcard selector instead, like I've got `getBySelLike` in DV.
Or if you insist on prefixing the id with `dashboarditem-` (which shouldn't really be necessary) you could create a new command that does this for you.

Reply:
I'll revisit this in an upcoming PR.